apps/prairielearn/assets/scripts/instructorAssessmentManualGradingInstanceQuestion.js

```diff
@@ -206,19 +218,26 @@ function updateSettingsPointValues() {
 
 function checkRubricItemTotals() {
   const form = document.querySelector('.js-rubric-settings-modal form');
+  if (!form) {
+    return;
+  }
   const startingPoints = Number(form.querySelector('[name="starting_points"]:checked')?.value ?? 0);
   const [totalPositive, totalNegative] = Array.from(form.querySelectorAll('.js-rubric-item-points'))
     .map((input) => Number(input.value))
     .reduce(
       ([pos, neg], value) => (value > 0 ? [pos + value, neg] : [pos, neg + value]),
       [startingPoints, startingPoints],
     );
-  const minPoints = Number(form.querySelector('[name="min_points"]').value);
+  const minPoints = Number(form.querySelector('[name="min_points"]')?.value ?? 0);
   const maxPoints =
-    Number(form.querySelector('[name="max_extra_points"]').value) +
-    Number(form.querySelector('.js-negative-grading').value);
-  form.querySelector('.js-settings-points-warning-placeholder').innerHTML = '';
-
+    Number(form.querySelector('[name="max_extra_points"]')?.value ?? 0) +
+    Number(form.querySelector('.js-negative-grading')?.value ?? 0);
+  const settingsPointsWarningPlaceholder = form.querySelector(
+    '.js-settings-points-warning-placeholder',
+  );
+  if (settingsPointsWarningPlaceholder) {
+    settingsPointsWarningPlaceholder.innerHTML = '';
+  }
```

Comment:
Once again wishing we had syntax like `foo?.bar = 'baz'`.

Reply:
Same 😄